config/locales/en/helpers.yml

```diff
@@ -246,7 +246,7 @@ en:
           military: For example, a dismissal or service detention
           prevention_reparation: For example, a restraining order or sexual harm prevention order
           # adults
-          adult_community_reparation: For example, a curfew, unpaid work or a restraining order. You might have been asked to wear a tag as part of your order
+          adult_community_reparation: For example, a sexual prevention order or a restraining order. You might have been asked to wear a tag as part of your order
```

Comment:
Probably better to use the complete "legal" term here, as we do already for under 18s: "sexual harm prevention order", just to be consistent.
https://hub.unlock.org.uk/knowledgebase/sexual-harm-prevention-order-shpo/

Reply:
Hmm, I've seen we use in the radios just Sexual order. So maybe it is ok to leave it just like that in the hint text too. We seem to have a mixture of terms though, would be good to standardised them.

<img width="323" alt="Screen Shot 2020-04-07 at 09 06 39" src="https://user-images.githubusercontent.com/687910/78645304-41fe0d80-78af-11ea-8f2e-450e214c2e96.png">